dev-libs/zziplib/zziplib-0.13.59.recipe

```diff
@@ -1,31 +1,95 @@
-DESCRIPTION="
-Lightweight library used to easily extract data from files archived in a \
-single zip file.
-"
+SUMMARY="The zziplib extract data from archives in a single zip"
+DESCRIPTION="Lightweight library used to easily extract data from \
+files archived in a single zip file."
 HOMEPAGE="http://zziplib.sourceforge.net/"
+COPYRIGHT="1999-2010 Guido Draheim"
+LICENSE="GNU LGPL v2.1
+	MPL v1.1"
 SOURCE_URI="http://sourceforge.net/projects/zziplib/files/zziplib13/0.13.59/zziplib-0.13.59.tar.bz2/download"
```

Comment:
OK, thanks! Now I see the second change (SOURCE_URI in zziplib-0.13.59.recipe.)
(I'll check later if everything is OK.)